src/itest/kotlin/nl/info/zac/itest/config/ProjectConfig.kt

```diff
@@ -110,9 +110,9 @@ class ProjectConfig : AbstractProjectConfig() {
                     headers = Headers.headersOf("Content-Type", "application/json"),
                     url = ZAC_HEALTH_READY_URL,
                     addAuthorizationHeader = false
-                ).use { response ->
+                ).let { response ->
```

Comment:
I think 'run' is better in this case, as the return value from `let` is not used.